test/extended/fixtures/ldap/ldapserver-deploymentconfig.json

```diff
@@ -58,6 +58,13 @@
             "securityContext": {
               "capabilities": {},
               "privileged": false
+            },
+            "readinessProbe": {
+              "exec": {
+                "command": ["ldapsearch", "-x", "-b", "dc=example,dc=com"]
```

Comment:
This seems to mark the container ready before the server is ready to receive requests. @liggitt any thoughts?